akka-persistence-testkit/src/main/scala/akka/persistence/testkit/internal/EventSourcedBehaviorTestKitImpl.scala

```diff
@@ -95,6 +97,9 @@ import akka.stream.scaladsl.Sink
   override val persistenceTestKit: PersistenceTestKit = PersistenceTestKit(system)
   persistenceTestKit.clearAll()
 
+  override val snapshotTestKit: SnapshotTestKit = SnapshotTestKit(system)
+  snapshotTestKit.clearAll()
```

Comment:
This means the SnapshotTestKit will always be used for all tests, also apps not using snapshots, I'm not sure about all of the consequences of this but I wonder if it wouldn't be better to do that more lazily only when `initialize` is actually used?

Reply:
`SnapshotTestKit` could be an `Option` here, present and used only if `akka.persistence.snapshot-store.plugin` in config is not empty, how about that?

Reply:
Sounds good!